src/app/accounts/platforms.ts

```diff
@@ -105,36 +105,14 @@ function loadPlatforms(membershipId: string): ThunkResult<readonly DestinyAccoun
   };
 }
 
-function loadActivePlatform(): ThunkResult<DestinyAccount | undefined> {
-  return async (_dispatch, getState) => {
-    const account = currentAccountSelector(getState());
-    if (account) {
-      return account;
-    }
-
-    const accounts = accountsSelector(getState());
-    if (!accounts.length) {
-      return undefined;
-    }
-
-    const membershipId = localStorage.getItem('dim-last-membership-id');
-    const destinyVersionStr = localStorage.getItem('dim-last-destiny-version');
-    const destinyVersion = destinyVersionStr ? parseInt(destinyVersionStr, 10) : 2;
-
-    const active = accounts.find(
-      (account) =>
-        account.membershipId === membershipId && account.destinyVersion === destinyVersion
-    );
-
-    return active ?? _.maxBy(accounts, (account) => account.lastPlayed);
-  };
-}
-
 export function logOut(): ThunkResult {
   return async (dispatch) => {
     removeToken();
     deleteDimApiToken();
+    localStorage.removeItem('dim-last-membership-id');
+    localStorage.removeItem('dim-last-destiny-version');
```

Comment:
We weren't clearing these out on logout, which would interact badly with the new "always load that account" logic.